fvp.mk

```diff
@@ -337,4 +338,5 @@ FVP_ARGS ?= \
 endif
 
 run-only:
-	$(FVP_PATH)/$(FVP_BIN) $(FVP_ARGS) $(FVP_EXTRA_ARGS)
+	make -p $(BINARIES_PATH)
```

Comment:
The `run-only` target should only run, not have any other side effect than what possibly the emulation itself.

Reply:
Updated it. I also changed the solution a bit to use "semihosting-cwd" instead of CD-ing into the folder. This seems to be a nicer way for our change.